eq/client/channel.cpp

```diff
@@ -88,7 +88,7 @@ using detail::STATE_FAILED;
 
 Channel::Channel( Window* parent )
         : Super( parent )
-        , _impl( new detail::Channel )
+        , _impl( new detail::Channel( *this ))
```

Comment:
iirc yes - but you don't seem to need it. frameViewFinish() already hast eq::Channel* as param.